docs/RFCS/README.md

```diff
@@ -110,25 +141,13 @@ conditions apply:
    expert is not available during the initial review (e.g. because she
    is on vacation).
 
-7. Once the changes described in the RFC have been made, change the
+9. Once the changes described in the RFC have been made, change the
    status of the PR from `in-progress` to `completed`. If subsequent
    developments render an RFC obsolete, set its status to `obsolete`.
 
 When you mark a RFC as obsolete, ensure that its text references the
 other RFCs or PRs that make it obsolete.
 
-# Rationale about minimum delays and process complexity
```

Comment:
I think this section should stay to motivate the choices made in the rest of the process.

Reply:
Leaving in with some modifications